SU2_CFD/include/solvers/CTurbSSTSolver.hpp

```diff
@@ -248,6 +248,26 @@ class CTurbSSTSolver final : public CTurbSolver {
                  CConfig *config,
                  unsigned short val_marker) override;
 
+  /*!
+   * \brief Impose the fluid interface boundary condition using tranfer data.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] conv_numerics - Description of the numerical method.
+   * \param[in] visc_numerics - Description of the numerical method.
+   * \param[in] config - Definition of the particular problem.
+   */
+  void BC_Fluid_Interface(CGeometry *geometry,
+                          CSolver **solver_container,
+                          CNumerics *conv_numerics,
+                          CNumerics *visc_numerics,
+                          CConfig *config) final {
+    BC_Fluid_Interface_impl(
+      [&](unsigned long iPoint) {
+        visc_numerics->SetF1blending(nodes->GetF1blending(iPoint), nodes->GetF1blending(iPoint));
+      },
+      geometry, solver_container, conv_numerics, visc_numerics, config);
+  }
```

Comment:
I don't get the difference in implementation between turbulence and species. Why is F1-blending implemented like this, but the diffusion coefficient is set in CSpeciesSolver::Viscous_Residual?

Problem is that it now crashes in species_diffusion.hpp in Finishresidualcalc because Diffusion_Coeff_i[iVar] is not initialized? 

Should there also be a BC_Fluid_Interface for SpeciesSolver, with SetDiffusionCoeff? At least this seems to solve the crash, but I'm not sure if this is then what you had in mind with this implementation